lua/lspconfig/server_configurations/pasls.lua

```diff
@@ -4,7 +4,7 @@ return {
   default_config = {
     cmd = { 'pasls' },
     filetypes = { 'pascal' },
-    root_dir = util.find_git_ancestor,
+    root_dir = root_pattern('*.lpi', '*.lpk', '.git'),
```

Comment:
you need to bring root pattern in scope, this fails lint because you are accessing an undefined function (root_pattern)

Reply:
@BarrOff to fix this, add the appropriate require() at top of the file. see other configs for example